packages/client/src/modals/Transaction/BannerInfo.js

```diff
@@ -22,7 +22,7 @@ const BannerInfo = ({
   const { displayName, tokenType } = getTokenMeta(vaultId);
   const { contractName: NFTName, contractAddress: NFTAddress } =
     parseIdentifier(collectionId);
-  const { name: imageName, thumbnail: imageURI } = nftMeta || {};
+  const { name: imageName, imageURI } = nftMeta || {};
```

Comment:
this should conform to `MetadataViews.cdc` and use `thumbnail` like in this PR: https://github.com/DapperCollectives/VESSEL/pull/131

Reply:
It doesn't work if I change it to thumbnail, the banner doesn't show an image.
If we make it work for ExampleNFT, how will it behave on testnet for all the NFTs that are deployed there?
Examples can be found here: https://nft-catalog.vercel.app/catalog/testnet/Momentables

Reply:
@KsenijaZivkovic this should work for both I think:

`const imageURI = nftMeta?.imageURI ?? nftMeta?.thumbnail;`